src/subscription_manager/cli_command/identity.py

```diff
@@ -87,15 +88,18 @@ def _do_command(self):
                 supported_resources = get_supported_resources(self.cp, self.identity)
                 if 'environments' in supported_resources:
                     consumer = self.cp.getConsumer(consumerid)
-                    environments = consumer['environments']
+                    evn_key = 'environments' if self.cp.has_capability(MULTI_ENV) else 'environment'
+                    environments = consumer[evn_key]
                     if environments:
-                        environment_name = (','.join([environment['name'] for environment in environments]))
+                        if evn_key == 'environment':
+                            environment_name = environments['name']
```

Comment:
It would be better to rename this variable `environment_name` to `environment_names`.